docs/docker.rst

```diff
@@ -22,65 +18,128 @@ Usage
+         $MILABENCH_IMAGE                          \
+         milabench report --runs /milabench/envs/runs
+
 
 ROCM
 ----
 
 Requirements
 ^^^^^^^^^^^^
 
-To run docker images you will need to install the softwares below.
-
 * rocm
 * docker
 
 Usage
 ^^^^^
 
+For ROCM the usage is similar to CUDA, but you must use a different image and the Docker options are a bit different:
+
 .. code-block:: bash
- 
-   sudo docker run -it  --rm --shm-size=1G                        \
-         --device=/dev/kfd --device=/dev/dri                      \
-         --security-opt seccomp=unconfined --group-add video      \
-         -v $(pwd)/results:/milabench/envs/runs                   \
-         ghcr.io/mila-iqia/milabench:rocm-nightly                 \
+
+   # Choose the image you want to use
+   export MILABENCH_IMAGE=ghcr.io/mila-iqia/milabench:rocm-nightly
+
+   # Pull the image we are going to run
+   docker pull $MILABENCH_IMAGE
+
+   # Run milabench
+   docker run -it --rm --shm-size=32G                        \
+         --device=/dev/kfd --device=/dev/dri                 \
+         --security-opt seccomp=unconfined --group-add video \
+         -v $(pwd)/results:/milabench/envs/runs              \
+         $MILABENCH_IMAGE                                    \
          milabench run
 
-   sudo docker run -it  --rm --shm-size=1G                        \
-         --device=/dev/kfd --device=/dev/dri                      \
-         --security-opt seccomp=unconfined --group-add video      \
-         -v $(pwd)/results:/milabench/envs/runs                   \
-         ghcr.io/mila-iqia/milabench:rocm-nightly                 \
-         milabench summary /milabench/envs/runs
+For the performance report, it is the same command:
+
+.. code-block:: bash
+
+   # Show Performance Report
+   docker run -it --rm                             \
+         -v $(pwd)/results:/milabench/envs/runs    \
+         $MILABENCH_IMAGE                          \
+         milabench report --runs /milabench/envs/runs
+
+
+Multi-node benchmark
+^^^^^^^^^^^^^^^^^^^^
+
+There is currently one multi-node benchmark, ``opt-2_7b-multinode``. Here is how to run it:
+
+* Provided you have the following two machines that can see each other on the network:
+  * ``manager-node``
+  * ``worker-node``
+* ``docker pull`` the image on both nodes.
+* Prior to running the benchmark, create a SSH key pair on ``manager-node`` and set up public key authentication to ``worker-node``.
+* On ``manager-node``, execute ``milabench run`` via Docker.
+  * Mount the private key at ``/milabench/id_milabench`` in the container
+  * Use ``--override`` statements as shown below to tell milabench about both nodes
+
+The command should look something like this:
 
+.. code-block:: bash
+
+    # On manager-node:
+
+    # Modify these variables to match your setup
+    export SSH_KEY=$HOME/.ssh/id_rsa
+    export NODE1=manager-node
+    export NODE2=worker-node
+    export NUM_MACHINES=2
```

Comment:
If you set NUM_MACHINES=4, then you need to have 3 entries in the worker list.  I don't know if it's clear here.